src/utils/utils.cpp

```diff
@@ -167,6 +170,33 @@ std::string mp::Utils::get_kernel_version() const
     return QSysInfo::kernelVersion().toStdString();
 }
 
+bool mp::Utils::client_certs_exist(const QString& cert_dir_path) const
+{
+    QDir cert_dir{cert_dir_path};
+
+    return cert_dir.exists(client_cert_file) && cert_dir.exists(client_key_file);
+}
+
+void mp::Utils::copy_client_certs_to_common_dir(const QString& cert_dir_path, const QString& common_cert_dir_path) const
+{
+    mp::utils::make_dir(common_cert_dir_path);
+    QDir common_dir{common_cert_dir_path}, cert_dir{cert_dir_path};
+
+    QFile::copy(cert_dir.filePath(client_cert_file), common_dir.filePath(client_cert_file));
+    QFile::copy(cert_dir.filePath(client_key_file), common_dir.filePath(client_key_file));
+}
```

Comment:
Are these utils? Feel more suited to `client_common`?

Reply:
Yeah, it can be moved back to `client_common`.  When I put it here, it was intended to also be used by the daemon in Windows when it imported the client cert, but since that solution doesn't need this, then I'll move it.

Reply:
This is now done.